src/pages/ProtocolPage.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import ProtocolOptions from "../components/ProtocolOptions";
+
+const styles = `
+    .row {
+        width: 100%;
+        height: 0vh;
+    }
+
+    .protocol-wrapper {
+        background-color: rgba(234, 234, 234, 1);
+        height: 100vh;
+    }
+
+    .protocol-number {
+        background-color: rgba(245, 148, 137, 1);
+        max-width: 85px;
+    }
+
+    .input-name {
+        border: 0px;
+        width: 150px;
+    }
+
+    .font-barlow {
+        font-family: 'Barlow', sans-serif;
+    }
+`;
+
+function ProtocolPage(props) {
+    return ( 
+        <div>
+            <div className="protocol-wrapper d-flex px-4 py-4">
+                <div className="row align-items-center">
+                    <div className="col-3">
```

Comment:
```suggestion
                    <div className="col-3 ps-0">
```
- A col da esquerda tem padding-left, mas a da direita não tem padding-right (até tem, mas os componentes internos tão violando);
- Pra não deixar o componente torto pra direita, acho mais fácil tirar o padding esquerdo do que descobrir a razão do direito ser violado (até pra padronizar com os outros componentes do form, que não tem padding);

Reply:
Acabei diminuindo o tamanho dos componentes da coluna da direita e alterei os paddings e margens pra tentar padronizar, talvez agora esteja melhor :D